images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -257,4 +257,10 @@ function Get-DacFxVersion {
     cd "C:\Program Files\Microsoft SQL Server\150\DAC\bin\"
     $dacfxversion = (./sqlpackage.exe /version)
     return "DacFx $dacfxversion"
-}
\ No newline at end of file
+}
+
+function Get-SwigVersion {
+    (swig | Out-String) -match "swig (?<version>\d+\.\d+\.?\d*)" | Out-Null
```

Comment:
I think we need to invoke `swig -version` here

Reply:
Fixed